jib-core/src/main/java/com/google/cloud/tools/jib/cache/Caches.java

```diff
@@ -48,11 +81,17 @@ public Initializer setApplicationCacheDirectory(Path applicationCacheDirectory)
       return this;
     }
 
-    public Caches init() throws CacheMetadataCorruptedException, NotDirectoryException {
-      if (baseCacheDirectory == null || applicationCacheDirectory == null) {
+    public Caches init()
+        throws CacheMetadataCorruptedException, IOException, CacheDirectoryNotOwnedException {
+      if (applicationCacheDirectory == null) {
         throw new IllegalStateException(
-            "Must initialize cache with both base image layer cache directory and application image layer cache directory");
+            "Must initialize cache with an application image layer cache directory");
+      }
+
+      if (DEFAULT_BASE_CACHE_DIRECTORY.equals(baseCacheDirectory)) {
+        ensureOwnership(DEFAULT_BASE_CACHE_DIRECTORY);
```

Comment:
How does the user deal with this, if this does indeed happen? They must go delete the cache, or something?

Reply:
There are a few options:

1) Tell the user to delete that cache directory
2) Provide the user with a configuration option to set the base cache directory
3) Default to using the same cache directory as the application cache directory

(These options can also be combined.)